operator/deploy/olm-catalog/ibm-spectrum-scale-csi-operator/1.1.0/ibm-spectrum-scale-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -66,6 +66,12 @@ spec:
         kind: CSIScaleOperator
         name: csiscaleoperators.csi.ibm.com
         resources:
```

Comment:
Shouldn't we make these changes to 2.0.0 directory, because right now this change seems to be happening in 1.1.0 olm-catalog dir ? And looks like we have not made changes in this file to remove ansible container, not sure if there is any other reason here.

Reply:
This is actually a separate  issue from the ansible stuff. This is more for the services and replicasets. This change will be pulled into  2.0.0. This was more to get the travis  to pass  in the new version and I wasn't ready to author the new csv  for 2.0.0.

Reply:
@mew2057 Thanks, not an issue we do have one more PR for that.